src/skin/imgloader.cpp

```diff
@@ -25,6 +26,9 @@ QImage* ImgLoader::getImage(const QString& fileName, double scaleFactor) const {
         }
     }
     if (scaleFactor > 1.0) {
+#else
+    if (qApp->devicePixelRatio() > 1.0) {
```

Comment:
Qt5 provides environment variable based settings for scaling factor. The screenshot is captured with `QT_SCALE_FACTOR=0.75` (2x is too big for my screen).